### DISCUSSION TITLE
Support to view Links between spans in UI

### DISCUSSION THREAD

**Discussion by dittybijil** [SigNoz/signoz#608] [General]
Can anyone help me to see Links between spans(links field attached to span) in SigNoz UI? We can see the relation in database, but no seeing an option view it in UI.

**Comment by pranay01** [comment]
hey @dittybijil can you share more details on what use case are you trying to solve with this? Will help us understand this better

**Reply by dittybijil** [reply]
@pranay01 
In OpenTelemetry while creating spans, like attributes and events we have an option to create links.(This link will contain the reference of some other span).
When we send the telemetry data to SigNoz that relation is correctly showing in SigNoz database.  But in SigNoz UI we cant find any option to see this spans relation.

**Reply by pranay01** [reply]
@dittybijil 
So the current [Gantt Chart](https://signoz.io/docs/userguide/trace-details/#1-gantt-chart-view) view and [Flame graph](https://signoz.io/docs/userguide/trace-details/#2-flame-graph-view) view is created through this span relationship. Is that what you are looking for?

But as of now I don't think there is a way to see the span ID number explicitly in the UI for a span.

**Reply by dittybijil** [reply]
@pranay01 
My need is to see this in Gantt Chart and Flame graph view.  But at present it is showing only the parent child relation.

**Reply by dittybijil** [reply]
@pranay01 
Found the link relation in Flame graph view. It was not as we expect and we changed it as parent child relation.  So its working perfectly now.  Thank you so much for your support

**Reply by pranay01** [reply]
@dittybijil  Nice to hear that. If you can also share what changes you made - may be other members in the community who also are facing similar issue can benefit from it :)
